### PR TITLE
Add article for "osu! at UCSD" and upcoming LAN tournament

### PR DESCRIPTION
**Pull request by cl8n**

### CODE REVIEW

File: wiki/Community/Organisations/osu!_at_UCSD/en.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - San Diego
+  - school
+  - university
+---
+
+# osu! at UCSD
+
+**osu! at UCSD** is an osu! club run as a division of [Triton Gaming](https://discord.gg/tritongaming "Triton Gaming Discord server") at the University of California, San Diego (*UCSD*). Its goal is to give osu! players at or around UCSD a space to connect with each other and build a local community. They host regular meetups both in-person and online, as well as LAN and online osu! tournaments.
+
+See the [osu! at UCSD tournament index](/wiki/Tournaments/osu!_at_UCSD) for wiki articles about some of their tournaments.
```
Comment (Walavouchey) [inline wiki/Community/Organisations/osu!_at_UCSD/en.md:15]:
```suggestion
See the [osu! at UCSD tournament index page](/wiki/Tournaments/osu!_at_UCSD) for more information about previously held tournaments.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - TGEX
+  - TGEX2022
+  - UCSD
+---
+
+# Triton Gaming Expo 2022
+
+![Triton Gaming Expo 2022 announcement banner](https://cdn.discordapp.com/attachments/228229738181820417/971166619948699678/TGEX.png)
+
+[osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) is hosting an osu! LAN tournament at **Triton Gaming Expo 2022** (***TGEX 2022***) in collaboration with [Triton Gaming](https://discord.gg/tritongaming "Triton Gaming Discord server"). Attendance and tournament participation is open to everybody. The exact schedule and details for the tournament have not been released yet.
```
Comment (Walavouchey) [inline wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:16]:
```suggestion
**Triton Gaming Expo 2022** (***TGEX 2022***) is an osu! LAN tournament hosted by [osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) in collaboration with [Triton Gaming](https://discord.gg/tritongaming). Attendance and tournament participation is open to everybody. The exact schedule and details for the tournament have not been released yet.
```

File: wiki/Tournaments/osu!_at_UCSD/en.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# osu! at UCSD tournaments
+
+Index page for all of the osu! tournaments hosted by [osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) at the University of California, San Diego.
```
Comment (Walavouchey) [inline wiki/Tournaments/osu!_at_UCSD/en.md:3]:
```suggestion
Index page for all osu! tournaments hosted by [osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) at the University of California, San Diego.
```

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md (hunk 2 above)
Comment (cl8n) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:16]: the lan doesnt have any real title, it's just a thing they're doing at tgex

File: wiki/Community/Organisations/osu!_at_UCSD/en.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  - club
+  - college
+  - San Diego
+  - school
+  - university
+---
+
+# osu! at UCSD
+
+**osu! at UCSD** is an osu! club run as a division of [Triton Gaming](https://discord.gg/tritongaming "Triton Gaming Discord server") at the University of California, San Diego (*UCSD*). Its goal is to give osu! players at or around UCSD a space to connect with each other and build a local community. They host regular meetups both in-person and online, as well as LAN and online osu! tournaments.
```
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Community/Organisations/osu!_at_UCSD/en.md:13]: it gives more context than only "external link"

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md (hunk 2 above)
Comment (Walavouchey) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:16]: then why is the article called TGEX 2022 🤔 

maybe "TGEX 2022 osu! LAN tournament"?

File: wiki/Community/Organisations/osu!_at_UCSD/en.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Community/Organisations/osu!_at_UCSD/en.md:13]: the context for the link is usually indicated in the link text anyway, as in `[Triton Gaming Discord server](https://discord.gg/tritongaming)`, which to me seems like a better way to handle it anyway

[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Community/Organisations/osu!_at_UCSD/en.md:13]: yes but im not rly talking abt discord server here. maybe better to not link at all in this sentence if I can't find anything better?

[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Community/Organisations/osu!_at_UCSD/en.md:13]: maybe, perhaps put it in the link section

File: wiki/Community/Organisations/osu!_at_UCSD/en.md (hunk 1 above)
Comment (cl8n) [reply wiki/Community/Organisations/osu!_at_UCSD/en.md:15]: previously held => their ?
the one in this PR isn't previous :^)

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## TGEX details
+
+TGEX will take place on Sunday, May 29th at [Price Center](https://en.wikipedia.org/wiki/Price_Center "\"Price Center\" on Wikipedia") Ballroom West (rooms 215 and 216) at the University of California, San Diego (*UCSD*). There is no entry fee. The event is expected to run from 12:00 p.m. to 10:00 p.m.
```
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:20]: this is only rly relevant to people in the area so i think local format is ok. it's how ive done other lan event too

[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:20]: no >:(

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  - osu! UCSD
+  - San Diego
+  - TGEX
+  - TGEX2022
+  - UCSD
+---
+
+# Triton Gaming Expo 2022
+
+![Triton Gaming Expo 2022 announcement banner](https://cdn.discordapp.com/attachments/228229738181820417/971166619948699678/TGEX.png)
```
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:14]: this is a temporary image just to have something, when I have a good one later I'll add that to wiki files

File: wiki/Community/Organisations/osu!_at_UCSD/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Community/Organisations/osu!_at_UCSD/en.md:15]: mm ok

```suggestion
See the [osu! at UCSD tournament index page](/wiki/Tournaments/osu!_at_UCSD) for more information about their tournaments.
```

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:20]: do add timezone though (`UTC+x`)

File: wiki/Community/Organisations/osu!_at_UCSD/en.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+See the [osu! at UCSD tournament index](/wiki/Tournaments/osu!_at_UCSD) for wiki articles about some of their tournaments.
+
+## Links
+
+- **[Discord server](https://discord.gg/DqBVmv5Knx)**
+- [UCSD Student Organization listing](https://studentorg.ucsd.edu/Home/Details/14917)
```
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Community/Organisations/osu!_at_UCSD/en.md:21]: keeping order cuz knowing who the officers are doesn't matter that much and also just copying UCI article

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md (hunk 2 above)
Comment (cl8n) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md]: feels natural to me to call these types of events by their parents' names, e.g. Zotcade at UCI. if it matters to make it more precise u can change it tho.

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+[osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) is hosting an osu! LAN tournament at **Triton Gaming Expo 2022** (***TGEX 2022***) in collaboration with *Triton Gaming*. Attendance and tournament participation is open to everybody. The exact schedule and details for the tournament have not been released yet.
+
+## TGEX details
+
+TGEX will take place on Sunday, May 29th at [Price Center](https://en.wikipedia.org/wiki/Price_Center) at the University of California, San Diego (*UCSD*). The event is expected to run from 12:00 p.m. to 10:00 p.m (UTC-7).
```
Comment (Walavouchey) [inline wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:20]:
```suggestion
TGEX will take place on Sunday, May 29th at [Price Center](https://en.wikipedia.org/wiki/Price_Center) at the University of California, San Diego (*UCSD*). The event is expected to run from 12:00 p.m. to 10:00 p.m. (UTC-7).
```

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:14]: it's not about being temporary, it's about hotlinking an external image

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md (hunk 2 above)
Comment (Walavouchey) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md]: well, probably ok for now

File: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:14]: the image is also over 1 mb, so it'll fail to load

[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md:14]: they're all cached regardless and the 1mb thing is for github, not osu

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**